examples/dial/CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+# Copyright 2023 Viam Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+find_package(gRPC CONFIG REQUIRED)
```

Comment:
Specifics pending based on above comments, but should we add a minimum version requirement here (and in other examples)? Attempting to build here produces a warning to that effect.

Reply:
Yes, definitely. There is a TODO to that effect up in the top-level `CMakeLists.txt`: https://github.com/viamrobotics/viam-cpp-sdk/pull/37/files#diff-1e7de1ae2d059d21e1dd75d5812d5a34b0222cef273b7c3a2af62eb747f9d20aR20.